.github/workflows/run_test_case.yaml

```diff
@@ -22,6 +22,8 @@ jobs:
         - uses: actions/checkout@v1
         - name: Code dialyzer
           run: |
+            # check validity of appup file
+            erl -noshell -eval "{ok,_}=file:consult('src/esockd.appup.src')" -s init stop
```

Comment:
Should be: `erl -noshell -eval "{ok,_}=file:script('src/esockd.appup.src', [{'VSN', "dummy_app_vsn"}])" -s init stop`